snippets/resolver/named.conf

```
@@ -1,28 +1,48 @@
 options {
-  directory "/var/cache/bind";
-  listen-on { any; };
-  listen-on-v6 { any; };
-  pid-file "/run/named/named.pid";
+directory "/var/cache/bind";
+allow-recursion {
+192.168.0.0/24;
+127.0.0.1/32;
 };
-
-zone "2tm1-admin-sys2.ephec.austreelis.net" IN {
-  type master;
-  file "/etc/bind/2tm1-admin-sys2.ephec.austreelis.net.zone";
+// Configure the IPs to listen on here.
+listen-on port 53 {any;};
+listen-on-v6 { none; };
+pid-file "/var/run/named/named.pid";
+allow-transfer { none; };
+};
+zone "." IN {
+type hint;
+file "/usr/share/dns/root.hints";
+};
```

Fix indentation:

```suggestion
zone "." IN {
  type hint;
  file "/usr/share/dns/root.hints";
};

```

Also, that zone file isn't included in the docker image, which is probably one of the reason why name resolution doesn't work for generic addresses.

---

I don't know how it is possible but this file is present in my docker image. So, I think the problem is not there